src/components/screens/hwWalletLogin/selectAccount/selectAccount.js

```diff
@@ -134,63 +117,53 @@ class SelectAccount extends React.Component {
 
   render() {
     const { t, device } = this.props;
-    const { hwAccounts } = this.state;
-
-    const {
-      nonEmptyAccounts,
-      emptyAccounts,
-      reclaimAccounts,
-    } = hwAccounts.reduce((acc, account, index) => {
-      if (account.legacy) {
-        acc.reclaimAccounts = [...acc.reclaimAccounts, { index, account }];
-        return acc;
-      }
-      if (account.summary?.balance > 0) {
-        acc.nonEmptyAccounts = [...acc.nonEmptyAccounts, { index, account }];
-        return acc;
-      }
-      acc.emptyAccounts = [...acc.emptyAccounts, { index, account }];
-      return acc;
-    }, { nonEmptyAccounts: [], emptyAccounts: [], reclaimAccounts: [] });
+    const { hwAccounts, hideEmptyAccounts } = this.state;
 
     return (
-      <div>
+      <div className={styles.selectAccountWrapper}>
         <h1>{t('Lisk accounts on {{WalletModel}}', { WalletModel: device.model })}</h1>
         <p>
           {t('Please select the account you’d like to sign in to or')}
           <TertiaryButton
             className={`${styles.createAccountBtn} create-account`}
             onClick={this.onAddNewAccount}
+            disabled={hwAccounts.some(account => account.summary?.balance === 0)}
```

Comment:
In case of hardware wallets the last account balance will always be zero and the `create account` button will be disabled all the time, should we consider not showing it if its not functional? @isalga @reyraa

Reply:
Agree it should not be displayed.